solr/core/src/java/org/apache/solr/core/FileSystemConfigSetService.java

```diff
@@ -238,9 +238,11 @@ public List<String> getAllConfigFiles(String configName) throws IOException {
           @Override
           public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
               throws IOException {
-            // don't include hidden (.) files
-            if (!Files.isHidden(file)) {
-              filePaths.add(configDir.relativize(file).toString());
+            String filePath = configDir.relativize(file).toString();
+            filePath = filePath.replace('\\', '/'); // normalize slashes
```

Comment:
I think that backslash is `path.getFileSystem().getSeparator()`.  In theory, we could be running on some new system that uses some other symbol and this method would return it; it'd be found in the path.toString as the separator.